packages/itmat-cores/src/GraphQLCore/studyCore.ts

```diff
@@ -409,10 +409,10 @@ export class StudyCore {
             // unversioned data: metadatafilter for versioned data and all unversioned tags
             if (versionId === null && aggregatedPermissions.hasVersioned) {
                 availableDataVersions.push(null);
-                fieldRecords = await this.db.collections.field_dictionary_collection.aggregate<IField>([{
-                    $match: { studyId: studyId, dateDeleted: null, dataVersion: { $in: availableDataVersions } }
+                fieldRecords = (await this.db.collections.field_dictionary_collection.aggregate<IField>([{
+                    $match: { studyId: studyId, dataVersion: { $in: availableDataVersions } }
                 }, {
-                    $sort: { dateAdded: -1 }
+                    $sort: { 'life.createdTime': -1 }
```

Comment:
Do you maintain a list of all the variable name chang/es somewhere? So that we can cross-check when we get to the migration script